### PR TITLE
Create next version of rancher-k3s-upgrader chart

### PR DESCRIPTION
**Pull request by thatmidwesterncoder**
Tracking issue: https://github.com/rancher/rancher/issues/41791

Creating a new chart and bumping the system-upgrade-controller to v0.13.1.

### CODE REVIEW

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
Can you split changes into two commits to ease the review: 1st for the base copy, and 2nd for changes?

File: charts/rancher-k3s-upgrader/0.6.0/Chart.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@ apiVersion: v1
...
   Users do not need to manually upgrade this app. It will be automatically upgraded to the latest version when upgrading a cluster.
-home: https://github.com/rancher/system-charts/blob/dev-v2.7/charts/rancher-k3s-upgrader
+home: https://github.com/rancher/system-charts/blob/dev-v2.8/charts/rancher-k3s-upgrader
 sources:
-  - "https://github.com/rancher/system-charts/blob/dev-v2.7/charts/rancher-k3s-upgrader"
-version: 0.5.0
-appVersion: v0.11.0
+  - "https://github.com/rancher/system-charts/blob/dev-v2.8/charts/rancher-k3s-upgrader"
+version: 0.6.0
+appVersion: v0.12.0
```
Comment (jiaqiluo) [inline charts/rancher-k3s-upgrader/0.6.0/Chart.yaml]: appVersion should match the image tag  v0.13.1 .

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — APPROVED** [review]

File: charts/rancher-k3s-upgrader/0.6.0/Chart.yaml [hunk 2]
```diff
@@ -2,9 +2,9 @@ apiVersion: v1
...
-home: https://github.com/rancher/system-charts/blob/dev-v2.7/charts/rancher-k3s-upgrader
+home: https://github.com/rancher/system-charts/blob/dev-v2.8/charts/rancher-k3s-upgrader
 sources:
-  - "https://github.com/rancher/system-charts/blob/dev-v2.7/charts/rancher-k3s-upgrader"
-version: 0.5.0
-appVersion: v0.11.0
+  - "https://github.com/rancher/system-charts/blob/dev-v2.8/charts/rancher-k3s-upgrader"
+version: 0.6.0
+appVersion: v0.13.1
 kubeVersion: '>= 1.23.0-0'
```
Comment (a-blender) [inline charts/rancher-k3s-upgrader/0.6.0/Chart.yaml:10]: The dev-v2.8 chart branch still supports 1.23 clusters?

Comment (thatmidwesterncoder) [reply charts/rancher-k3s-upgrader/0.6.0/Chart.yaml:10]: Apparently? haha. Is there a doc somewhere that says what we support as far as the upstream cluster version? I'm happy to bump it up if we don't support 1.23 anymore.

Comment (a-blender) [reply charts/rancher-k3s-upgrader/0.6.0/Chart.yaml:10]: @jiaqiluo @aiyengar2 Lucking out on finding docs. Are there docs that outline what k8s versions system-charts are supported on? Just all versions of Rancher that are actively being maintained, which excludes v2.5 and goes down to 1.23? If not now, support will be removed soon I think.

Comment (a-blender) [reply charts/rancher-k3s-upgrader/0.6.0/Chart.yaml:10]: From double checking in an offline discussion, SUC kube version needs to align with what k8s versions are supported by that Rancher version. We'll keep 1.23 support for Rancher 2.8 now but that will likely change in the near future as k8s 1.23 is approaching EoL.

PR lgtm

**Review by a-blender — APPROVED** [review]

**Pull request merged**